### PR TITLE
Feature/refactor scripts

### PR DESCRIPTION
**Pull request by Leasw144**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features
- [x] Refactoring
- [ ] Planning

### Detailed Description
- This PR seeks to combine the fetchData document back into scripts and clears the bugs associated within the code that prevented the instantiation of the appropriate data sets. In short, it looks closer to what we had earlier this week and will hopefully make things a bit clearer moving forward. Additionally, the variable `todayDate` was edited to todaysDate for semantic reasons across all relevant documents
### Why is this change required? What problem does it solve?
 - It was realized that the previous path we were on was somewhat harder to logic through so this was a move to get us back to something that was akin to what we had earlier.
### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
 - tons of bugs kept showing up that prevented our data from being properly implemented so certain functions and lines of code have been temporarily commented out. Methods that are meant to instantiate our data has also been added into to fetch method. It should be noted that we moved the fetchData contents back into scripts because we felt that the separation was having an additional adverse affect on how the engine was reading the code.
### Files modified:
- [ ] index.html
- [ ] .scss // file name(s):
- [x] classes // file name(s):
- [ ] test // files name(s):
- [x] scripts.js
- [x] Other files: fetchData

### CODE REVIEW

**Comment by leighlars** [comment]
Yes, lots of bugs, intentionally. Bad day of communication on my end, and I own that or else I would have been able to explain why they were there and my thoughts for resolving for them moving forward, maybe that could have saved you some time. I'm thankful you fixed them! What an undertaking.

File: src/fetchData.js [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-      return data.userData;
-    })
-    .catch(err => console.log(err.message))
+// function fetchData() {
+//   let userData =  fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData")
+//     .then(response => response.json())
+//     .then(data => {
+//       return data.userData;
+//     })
+//     .catch(err => console.log(err.message))
```
Comment (leighlars) [inline src/fetchData.js:8]: I think we should leave this file separate.

File: src/HydrationRepository.js [hunk 2]
```diff
@@ -5,10 +5,10 @@ class HydrationRepository {
     this.hydrationData = this.instantiateHydroData(rawHydrationData);
   }
 
-  instantiateHydroData(rawHydrationData) {
```
Comment (leighlars) [inline src/HydrationRepository.js:8]: What was the motivation to instantiate each hydration data when no methods or properties are added to it? Is this redundant?

File: src/fetchData.js (hunk 1 above)
Comment (NakiNorton) [reply src/fetchData.js:8]: We can definitely move it back to a separate file. We ran into issues with the chain breaking and not getting the data we wanted so we moved it onto the scripts file to debug and get a better understanding of the logic. It's much cleaner to have it on a separate file though.

**Comment by NakiNorton** [comment]
Thanks for all your work on this Linus! The scripts file is starting to come together and now we can start focusing on the classes and getting everything connected.
As Leigh suggested (and we also discussed) it'll be good to move the fetch data back to it's separate file now that we've got a better understanding of the fetch logic.

**Pull request merged**